src/puppetlabs/puppetdb/command/dlo.clj

```diff
@@ -6,16 +6,14 @@
    [puppetlabs.i18n.core :refer [trs]]
    [puppetlabs.kitchensink.core :refer [timestamp]]
    [puppetlabs.puppetdb.nio :refer [copts copt-atomic copt-replace oopts]]
-   [puppetlabs.puppetdb.queue :as queue
-    :refer [cmdref->entry metadata-str parse-cmd-filename]]
+   [puppetlabs.puppetdb.queue :as q]
+   [puppetlabs.puppetdb.utils :refer [utf8-length utf8-truncate]]
    [puppetlabs.stockpile.queue :as stock])
   (:import
    [java.nio.file AtomicMoveNotSupportedException
     FileAlreadyExistsException Files LinkOption]
    [java.nio.file.attribute FileAttribute]))
 
-(def command-names (cons "unknown" queue/metadata-command-names))
```

Comment:
Sorry, this one doesn't exactly, but the next one (#2067) adds stricter behavior around acceptable command names such that "unknown" will throw an error with the default queue metadata serializer.